### ISSUE TITLE
why is this published on nuget ?

### ISSUE THREAD

**Issue by benoitjadinon** [saadfarooq/Plugin.RxLocation#1]
why was this empty project published on nuget ? you just wasted my time. well done.

**Comment by saadfarooq** [comment]
I was just following a tutorial trying to figure it out but had to give up because it was taking too much time. I apologize to the entire world for the time you wasted in going to a website a clicking a link and consequently the great opportunities for world peace that we missed as a consequence.

**Comment by benoitjadinon** [comment]
good you find this funny... I added the nuget dependency in every project of my solution, implemented the code everywhere, until the last moment when I noticed IRxLocation didn't have any property...
can't you just remove it from nuget ?

**Comment by benoitjadinon** [comment]
also, if I fork it and eventually implement it, would you share the nuget credentials ?

**Comment by saadfarooq** [comment]
A few comments here:
1. I don't like the idea of publishing in development nugets to nuget.org either but that's how the official Xamarin team recommended it. I'm more experienced in Java land and local Maven's are really easy to work with there. I eventually did find a way to do it locally but it was still pretty awful.
2. It's always useful to commit before adding an untested dependency I find. I imagine with the tutorial's recommendations there are plenty of unresolved projects on nuget (to be fair, that's probably true of all repositories).
3. I'm more happy to collaborate on this. The reason I stopped working on it was that it got to a state where it wasn't working as expected and debugging wasn't getting me into the correct implementations.

**Comment by benoitjadinon** [comment]
yes, there should totally be 1/5 stars notation of some sort on nuget libs. also, comments.
